Zappy_IA_Report/DTC/Video/Tutorial2.py

```
@@ -0,0 +1,25 @@
+X1 = df.iloc[:,0:13]
+X2 = df.iloc[:,0:13]
+
+from sklearn.preprocessing import StandardScaler
+
+scaleStandard = StandardScaler()
+X1 = scaleStandard.fit_transform(X1)
+X1 = pd.DataFrame(X1, columns=['YRS', 'G', 'AB', 'R', 'H', '2B', '3B', 'HR', 'RBI','BB','SO', 'SB'])
+print(X1.head())
+print(X1.describe().round(3))
+
+from sklearn.preprocessing import MinMaxScaler
```

Ruff (0.11.9)

Module level import not at top of file

(E402)